internal/config/plugin.go

```diff
@@ -17,12 +17,13 @@ type Plugins struct {
 
 // Plugin describes a K9s plugin
 type Plugin struct {
-	ShortCut    string   `yaml:"shortCut"`
-	Scopes      []string `yaml:"scopes"`
-	Description string   `yaml:"description"`
-	Command     string   `yaml:"command"`
-	Background  bool     `yaml:"background"`
-	Args        []string `yaml:"args"`
+	ShortCut      string   `yaml:"shortCut"`
```

Comment:
I think we should just call the new option confirm as it will always appear as a dialog. Also we will need to update the Docs to define this new option.

Reply:
Besides editing readme should I also add `confirm: false` for plugins examples in [here](https://github.com/derailed/k9s/tree/master/plugins)?

Reply:
Yes I think for the ones that warrants it ex `job_suspend` otherwise the zero value will be used, which I think is fine. Thank you David!